components/Embed.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+interface Props {
+  src: string;
+  title?: string;
+  width?: number;
+  height?: number;
+}
+
+export function Embed({ src, title, width, height }: Props) {
+  return (
+    <iframe
+      referrerPolicy="no-referrer"
+      sandbox="allow-forms allow-scripts"
+      src={src}
+      title={title}
+      width={width ?? "100%"}
```

Comment:
**issue:** user shouldn't have to write percentage. We need the user to write just the number and then add %.

```typescript
width={`${width ?? 100}%`}
```

Reply:
praise: Makes sense!

Reply:
Great, can we fix it? :)